docs/map.jinja.rst

```diff
@@ -0,0 +1,330 @@
+.. _map.jinja:
+
+``map.jinja``: gather formula configuration values
+==================================================
+
+The `documentation`_ gives a quick view of the use of a ``map.jinja`` to gather parameters values for a formula.
+
+Current best practice is to let ``map.jinja`` handle parameters from all sources, to minimise the use of pillars, grains or configuration from ``sls`` files and templates directly.
+
+The ``map.jinja`` file uses several sources where to lookup parameter values:
+
+- Multiple YAML files organised in the `fileserver`_ under the formula ``parameters/`` directory
+- Configuration collected by `salt['config.get']`_
+
+As `pillars`_ are rendered on the Salt master for every minion, this increases the load on the master as the pillar values and the number of minions grows.
+
+As a good practice, you should:
+
+- store non-secret data in YAML files distributed by the `fileserver`_
+- store secret data in:
+
+  - `pillars`_ (and look for the use of something like `pillar.vault`_)
+  - `SDB`_ (and look for the use of something like `sdb.vault`_)
+
+
+.. contents:: **Table of Contents**
+
+For formula users
+-----------------
+
+Quick start: configure per DNS domain name values
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+We will see a quick setup to configure the ``TEMPLATE`` formula for different DNS domain name.
```

Comment:
s/TEMPLATE/libvirt

Reply:
I choose `TEMPLATE` for all formula to have the same file and ease the spread of modification by `ssf`.

Reply:
so it's supposed to be changed by the convert script or to stay with "TEMPLATE"?

Reply:
It's supposed to stay with `TEMPLATE` and now I wonder if it's possible without excluding that file from `convert-formula.sh` processing.